spec/ameba/rule/lint/literals_comparison_spec.cr

```diff
@@ -41,26 +41,13 @@ module Ameba::Rule::Lint
         CRYSTAL
     end
 
-    it "reports if there is a static path comparison evaluating to false" do
-      expect_issue subject, <<-CRYSTAL
-        String == Nil
-        # ^^^^^^^^^^^ error: Comparison always evaluates to false
-        CRYSTAL
-    end
-
     context "macro" do
-      pending "reports in macro scope" do
+      it "reports in macro scope" do
```

Comment:
could you please add a positive test in the macro scope? i.e. when there are no issues reported

Reply:
Sure, done.